tex/gsp-default.tex

```diff
@@ -221,12 +221,14 @@
 %
 % minimal space between letters of different syllable texts for text around bars
 % (new bar spacing algorithm only)
-% Note that the plus and minus values of interwordspacetext@bars or interwordspacetext@bars@euouae
-% are what will be used around bars in the new bar spacing algorithm
 \grecreatedim{interwordspacetext@bars}{0.15 cm}{scalable}%
 % minimal space between letters of different syllable texts for text around bars,
 % euouae context
 \grecreatedim{interwordspacetext@bars@euouae}{0.15 cm}{scalable}%
+\grecreatedim{interwordspacetext@bars@notext}{0.3 cm}{scalable}%
+% minimal space between letters of different syllable texts for text around bars,
+% euouae context
+\grecreatedim{interwordspacetext@bars@notext@euouae}{0.3 cm}{scalable}%
```

Comment:
I wound tend to make it a big bigger, something like 0.35 or even 0.4... But it's not that important, the main issue is that it's now possible to tune it!